crates/rome_service/src/configuration/javascript.rs

```diff
@@ -98,3 +101,17 @@ impl Default for PlainQuoteStyle {
         Self::Double
     }
 }
+
+#[derive(Deserialize, Serialize, Debug, Eq, PartialEq)]
+#[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
+#[serde(rename_all = "camelCase", remote = "QuoteProperties")]
+pub enum PlainQuoteProperties {
+    AsNeeded,
+    Preserve,
+}
+
+impl Default for PlainQuoteProperties {
+    fn default() -> Self {
+        Self::AsNeeded
+    }
+}
```

Comment:
```suggestion
#[derive(Deserialize, Default, Serialize, Debug, Eq, PartialEq)]
#[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
#[serde(rename_all = "camelCase", remote = "QuoteProperties")]
pub enum PlainQuoteProperties {
    #[default]
    AsNeeded,
    Preserve,
}
```

We can now use macros to assign defaults to enum!

Reply:
Cool 🦀

Reply:
It may be better to call `QuoteStyle::default` here to make sure these two are always in sync. Not sure if there's a `From` implementation that easily allows you to do so.

Reply:
Sorry, I'm not sure, please tell me how to do it